lib/editing/editor_codemirror.dart

```diff
@@ -535,7 +535,8 @@ class _CodeMirrorDocument extends Document<_CodeMirrorEditor> {
   @override
   void setAnnotations(List<Annotation> annotations) {
     for (final marker in doc.getAllMarks()) {
-      if (marker.jsProxy != null && marker.jsProxy!['atomic'] == false) {
+      if (marker.jsProxy != null &&
+          (marker.jsProxy!['atomic'] ?? false) == false) {
```

Comment:
I think this could also be `marker.jsProxy!['atomic'] != true`? Happy to update if you prefer that.

Reply:
`marker.jsProxy!['atomic'] != true` is probably much easier to read and will work also.  

I did not take into account the 'atomic' property not existing when adding this code.  Checking for it not to be true should work fix this bug also (and also handle the case where the property does not exist). (Because what I was actually doing here is trying to exclude code expand/collapse markers, and they will always have 'atomic' set explicitly to true).

Thanks@jakemac53 for finding and fixing this bug (that I inadvertently created 😞 ) !

Reply:
Changed this to != true as I agree its easier to read